pkg/ruler/notifier.go

```diff
@@ -150,19 +164,34 @@ func amConfigFromURL(rulerConfig *Config, url *url.URL, apiVersion config.Alertm
 		PathPrefix:              url.Path,
 		Timeout:                 model.Duration(rulerConfig.NotificationTimeout),
 		ServiceDiscoveryConfigs: sdConfig,
+		HTTPClientConfig: config_util.HTTPClientConfig{
+			TLSConfig: config_util.TLSConfig{
```

Comment:
https://github.com/cortexproject/cortex/pull/3156 is close to get merged. I would wait for it and add `ServerName` support too.

Reply:
Ignore the mention about #3156.